frontend/src/common/queries/filter.ts

```diff
@@ -220,6 +220,51 @@ export function useFetchDatasetRows(): FetchCategoriesRows<DatasetRow> {
   };
 }
 
+interface CorpusSummary {
+  cellCount: number;
+  cellTypesCount: number;
+  datasets: number;
+  isError: boolean;
+  isLoading: boolean;
+}
+
+export function useCorpusSummary(): CorpusSummary {
+  const datasetRows = useFetchDatasetRows();
+  const summary = datasetRows.rows.reduce(
+    (acc, row) => {
+      const newCellTypes = row.cell_type.reduce((acc1, cellType) => {
+        if (acc.cellTypes.indexOf(cellType.ontology_term_id) === -1) {
+          acc1.push(cellType.ontology_term_id);
+        }
+        return acc1;
+      }, new Array<string>());
```

Comment:
let's avoid using `new Array()` 🙏 

https://coderwall.com/p/h4xm0w/why-never-use-new-array-in-javascript
```suggestion
      }, [] as string[]);
```